docs/examples/host-guest-implicit.rst

```diff
@@ -0,0 +1,125 @@
+.. _host_guest_implicit:
+
+Absolute Binding of Host cucurbit[7]uril with Guest molecule B2 in Implicit Solvent
+===================================================================================
+
+This example illustrates the computation of the binding free energy of B2 guest to the host cucurbit[7]uril in an
```

Comment:
We should link to [this AMBER tutorial](http://ambermd.org/tutorials/advanced/tutorial21/) that was the source of files for CB7 and B2.